tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -547,7 +564,7 @@ def log_call_i(x):
         return log(x)
 
     f1 = epyccel(log_call_i, language = language)
-    x = randint(low=min_float, high=1e6)
+    x = randint(low=0, high=1e6)
     assert(isclose(f1(x), log_call_i(x), rtol=RTOL, atol=ATOL))
```

Comment:
For the logarithm `0` is not an acceptable argument, so you should start from `1` in this case.